src/ActorMultiVertex.cpp

```diff
@@ -1051,7 +1051,7 @@ class LunaActorMultiVertex: public Luna<ActorMultiVertex>
 		lua_createtable(L, 4, 0);
 		lua_pushnumber(L, state.rect.left * width_ratio);
```

Comment:
Hmm why dont these others need to be casted too?

Reply:
https://github.com/itgmania/itgmania/pull/852/files#r2106305763